### DISCUSSION TITLE
Passing data to the editor while Save

### DISCUSSION THREAD

**Discussion by shadhinplanet** [jeffreyvr/dropblockeditor#28] [Q&A]
I have created several elements. Now i want to store output data in my database table with a relationship for that specific page. So i need to pass a data/id to the editor while saving the data. I mean the SAVE Button. also need to load data/elements while edit the page. I need help to do it.

**Comment by jeffreyvr** [comment]
Hi, you can extend the `DropBlockEditor ` component class and then add your desired data.

Here is an example of how this was done for MailCoach:
https://github.com/jeffreyvr/dropblockeditor-mailcoach/blob/main/src/Http/Livewire/Editor.php

And the SaveButton:
https://github.com/jeffreyvr/dropblockeditor-mailcoach/blob/main/src/Http/Livewire/SaveButton.php

Will try to add some more info on this in the documentation.
